delivery_postlogistics/models/stock_picking.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+from operator import attrgetter
+
+from odoo import _, exceptions, fields, models
+
+from ..postlogistics.web_service import PostlogisticsWebService
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    delivery_fixed_date = fields.Date(
+        "Fixed delivery date", help="Specific delivery date (ZAW3217)"
+    )
+
+    # TODO: consider refactoring these fields using a partner relation instead
+    delivery_place = fields.Char(
+        "Delivery Place", help="For Deposit item service (ZAW3219)"
+    )
+    delivery_phone = fields.Char(
+        "Phone", help="For notify delivery by telephone (ZAW3213)"
+    )
+    delivery_mobile = fields.Char(
+        "Mobile", help="For notify delivery by telephone (ZAW3213)"
+    )
+
+    def _get_packages_from_picking(self):
+        """ Get all the packages from the picking """
+        self.ensure_one()
+        operation_obj = self.env["stock.move.line"]
+        operations = operation_obj.search(
+            [
+                "|",
+                ("package_id", "!=", False),
+                ("result_package_id", "!=", False),
+                ("picking_id", "=", self.id),
+            ]
+        )
+        package_ids = []
```

Comment:
:fire:
```suggestion
        package_ids = set([])
```

Reply:
https://github.com/trobz/delivery-carrier/pull/5

Reply:
@sebalix please redo your PR on main branch as this was prematurely merged